src/main.cpp

```diff
@@ -2991,10 +2991,7 @@ void vcMain_ShowLoginWindow(vcState *pProgramState)
     {
       ImGui::TextUnformatted(vcString::Get(loginStatusKeys[pProgramState->loginStatus]));
 
-      const bool isErrorStatus = udStrBeginsWith(loginStatusKeys[pProgramState->loginStatus], "loginError");
-
-      if (isErrorStatus)
-        ImGui::PopStyleColor();
+      udStrBeginsWith(loginStatusKeys[pProgramState->loginStatus], "loginError");
```

Comment:
This doesn't do anything, just remove it?

Reply:
True, I just saw that the error was unnecessary, I did not check the function.

Reply:
Removed